include/bl_validation.rst

```diff
@@ -0,0 +1,28 @@
+.. _doc_bl_validation:
+
+Bootloader firmware validation
+##############################
+
+The bootloader firmware validation library provides the function that the :ref:`bootloader` uses to validate a firmware image before booting it.
+
+The API is public because applications that are booted by the immutable bootloader can call the function from this library via the bootloader's code, through external APIs.
+See :ref:`doc_fw_info_ext_api` for more information.
+Using this mechanism can be useful when the application receives a DFU package and wants to determine whether it will be accepted by the bootloader.
+
+Validation
+**********
+
+The :cpp:func:`bl_validate_firmware` function validates the following information:
```

Comment:
The link to the function doesn't work (same for line 18) - probably because it's defined as EXT_API_FUNCTION.
Any idea how to fix that?

Reply:
Is code preprocessed in any way before building the docs?

Reply:
There's the usual doxygen config file at nrf/doc/nrf/nrf.doxyfile.in

Reply:
I removed the EXT_API_FUNCTION macro, so now it's just a regular function declaration.